install/upgrade_to_2_0.php

```diff
@@ -355,6 +355,32 @@ public function upgrade(Migration $migration) {
       $table = 'glpi_plugin_flyvemdm_mqttupdatequeues';
       $migration->dropTable($table);
 
+      $table = 'glpi_plugin_flyvemdm_mqttlogs';
+      $migration->changeField($table, 'message', 'message', 'mediumtext');
+      $migration->addField($table, 'itemtype', 'string', ['after' => 'message']);
+      $migration->addField($table, 'items_id', 'integer', ['after' => 'itemtype']);
+      $migration->addKey($table, 'itemtype', 'itemtype');
+      $migration->addKey($table, 'items_id', 'items_id');
+      if (!$DB->fieldExists($table, 'name')) {
+         // fix fleets logs
```

Comment:
this is not a fix.. this is not a feature, this is .. an upgrade !

Reply:
I don't get the meaning for this comment and the next one, there is no going to be an action for migrate the data?

Reply:
this comment is about the "fix fleets logs" sentence. This code does not fixes a bug, it updates the content of a table to match a new specification for logs.

I think "fix" is not accurate here